Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/Modifier.h

```diff
@@ -29,6 +31,8 @@ namespace ScriptCanvasEditor
                 , AZStd::vector<SourceHandle>&& assets
                 , AZStd::function<void()> onComplete);
 
+            virtual ~Modifier();
```

Comment:
if the class is final, why does it need a virtual dtor?

Reply:
No reason, I will remove the virtual specifier.